backend/app/tests/services/llm/providers/test_google_ai.py

```
@@ -7,11 +7,18 @@
 import pytest
 import requests
 
+# ponytail: Vertex provider is temporarily disabled in the registry; skip only
+# the routing/execute tests. Pure client tests (endpoint, sa_info, credential
+# fallback) still run for coverage.
+_vertex_disabled = pytest.mark.skip(
+    reason="Vertex provider disabled — routing swapped to GoogleAIProvider"
+)
+
 from app.core.audio_utils import AudioRef
```

Ruff (0.16.1)

[error] Module level import not at top of file

(E402)